codexctl/updates.py

```diff
@@ -82,11 +83,21 @@ def get_remarkable_versions(self) -> tuple[dict, dict, dict, str, str]:
 
         self.logger.debug(f"Version ids contents are {contents}")
 
+        provider_urls = contents.get("external-provider-urls", contents.get("external-provider-url"))
+        if isinstance(provider_urls, str):
+            provider_urls = [provider_urls]
+        if provider_urls is None:
+            raise SystemError(
+                f"version-ids.json at {file_location} is missing external provider URLs. "
+                "Please delete the file and try again, or open an issue on the repo."
+            )

```

Comment:
```suggestion
        if provider_urls is None:
            raise SystemError(
                f"version-ids.json at {file_location} is missing external provider URLs. "
                "Please delete the file and try again, or open an issue on the repo."
            )

        if isinstance(provider_urls, str):
            provider_urls = [provider_urls]
```
provider_urls will never be None if it's a string, so moving the exception early reduces cognitive load while reading this code again in the future.